libs/sdk-ui-ext/src/internal/components/configurationControls/CheckboxControl.tsx

```diff
@@ -55,7 +55,11 @@ class CheckboxControl extends React.Component<ICheckboxControlProps & WrappedCom
         const { valuePath, properties, pushData } = this.props;
 
         const clonedProperties = cloneDeep(properties);
-        set(clonedProperties, `controls.${valuePath}`, event.target.checked);
+        set(
+            clonedProperties,
+            `controls.${valuePath}`,
+            valuePath === "disableDrillDown" ? !event.target.checked : event.target.checked,
```

Comment:
The component should imho not know about the valuePaths internals. 
I would rather add new optional prop to the component called `isValueInverted`, use this prop in this condition and in interaction section where the component is used set the prop to `true`. You don't need to update another occurrences of the Checkbox component as the prop would be optional and set to false by default for all these other instances.

Reply:
I have updated this. Thank you!